### PR TITLE
.NET Standard TraceListener logging

### PR DESCRIPTION
**Pull request by pharring**
Add a .NET Standard version of the TraceListener logging package so that it can be used in .NET Core applications.

### CODE REVIEW

File: src/TraceListener/AssemblyInfo.cs [hunk 1]
```diff
@@ -31,6 +31,7 @@
 [assembly: Guid("3aada9bf-fc15-42fd-ace5-4cb2f11c1f2a")]
 
 [assembly: InternalsVisibleTo("Microsoft.ApplicationInsights.TraceListener.Net45.Tests, PublicKey=" + AssemblyInfo.PublicKey)]
+[assembly: InternalsVisibleTo("Microsoft.ApplicationInsights.TraceListener.Tests, PublicKey=" + AssemblyInfo.PublicKey)]
```
Comment (SergeyKanzhelev) [inline src/TraceListener/AssemblyInfo.cs]: maybe have `netcoreapp10` in the name of test assembly?

File: test/TraceListener.netcoreapp10.Tests/TraceListener.netcoreapp10.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  <PropertyGroup>
+    <RootNamespace>Microsoft.ApplicationInsights.TraceListener.Tests</RootNamespace>
+    <AssemblyName>Microsoft.ApplicationInsights.TraceListener.Tests</AssemblyName>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\TraceListener.Net45.Tests\ApplicationInsightsTraceFilterTests.cs" Link="ApplicationInsightsTraceFilterTests.cs" />
```
Comment (SergeyKanzhelev) [inline test/TraceListener.netcoreapp10.Tests/TraceListener.netcoreapp10.Tests.csproj]: moving thos to the folder with the name Shared can make folders structure cleaner. Can you do it please?

File: src/TraceListener/AssemblyInfo.cs (hunk 1 above)
Comment (pharring) [reply src/TraceListener/AssemblyInfo.cs]: Done

File: test/TraceListener.netcoreapp10.Tests/TraceListener.netcoreapp10.Tests.csproj (hunk 2 above)
Comment (pharring) [reply test/TraceListener.netcoreapp10.Tests/TraceListener.netcoreapp10.Tests.csproj]: Done

**Comment by pharring** [comment]
Ready to merge (I don't have permission myself)

**Pull request merged**